requirements.txt

```diff
@@ -15,4 +15,4 @@ six
 applicationinsights
 commentjson
 pypiwin32==219; sys_platform == 'win32' and python_version < '3.6'
-pypiwin32==223; sys_platform == 'win32' and python_version >= '3.6'
\ No newline at end of file
+pypiwin32==223; sys_platform == 'win32' and python_version >= '3.6'
```

Comment:
Seems the strings are identical, what's changed?

Reply:
I just change `click` through GitHub web portal. I guess GitHub has do some adjustment for the line ending which should  not have side effect.